src/radical/pilot/session.py

```diff
@@ -509,11 +519,8 @@ def _init_cfg_from_registry(self):
                                         level=self._cfg.get('log_lvl'),
                                         debug=self._cfg.get('debug_lvl'))
 
-        from . import version_detail as rp_version_detail
-        self._log.info('radical.pilot version: %s', rp_version_detail)
-        self._log.info('radical.utils version: %s', ru.version_detail)
+        self._log_setup()
```

Comment:
is this method missing?

Reply:
I thought it was inherited. Yes, it is missing and needs to be defined.
```python
In [1]: import radical.pilot as rp

In [2]: rp.Session._log_setup
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[2], line 1
----> 1 rp.Session._log_setup

AttributeError: type object 'Session' has no attribute '_log_setup'
```

Reply:
Thanks for catching this - this should have been `self._log_version` and is fixed now.